test/test_data/libvncserver.py

```diff
@@ -9,30 +9,30 @@
     }
 ]
 package_test_data = [
+    {
+        "url": "http://archive.ubuntu.com/ubuntu/pool/main/libv/libvncserver/",
+        "package_name": "libvncserver1_0.9.10+dfsg-3ubuntu0.16.04.6_amd64.deb",
+        "product": "libvncserver",
+        "version": "0.9.10",
+    },
     {
         "url": "http://mirror.centos.org/centos/8/AppStream/x86_64/os/Packages/",
         "package_name": "libvncserver-0.9.11-17.el8.x86_64.rpm",
         "product": "libvncserver",
         "version": "0.9.11",
     },
     {
-        "url": "https://ftp.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/9.1/All/",
-        "package_name": "libVNCServer-0.9.13.tgz",
```

Comment:
I think we could've kept this file in the test instead of `.deb` since it's a different extension and we have another `.deb` file.